week3/webApplicationHw/Controllers/CarController.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.AspNetCore.Mvc;
+using webApplicationHw.Models;
+
+namespace webApplicationHw.Controllers;
+
+
+[ApiController]
+[Route("[controller]")]
+public class CarController : ControllerBase
+{
+
```

Comment:
Try avoiding new empty lines that are not used to delimit functionality or logic, same for lines: 5,38, 35, 41,48-50